src/storage/page_buffer.c

```diff
@@ -15158,7 +15218,7 @@ pgbuf_bcb_is_to_vacuum (const PGBUF_BCB * bcb)
 STATIC_INLINE bool
 pgbuf_bcb_avoid_victim (const PGBUF_BCB * bcb)
 {
-  return (bcb->flags & PGBUF_BCB_INVALID_VICTIM_CANDIDATE_MASK) != 0;
+  return pgbuf_bcb_flag_is_invalid_victim (bcb->flags);
```

Comment:
if functions are synonymous...?

Reply:
Yeah, almost :). The signature is a bit different.